index.html

```diff
@@ -3922,33 +3899,20 @@ <h2>
             <td>
               <p>If child of `role=grid`, `rowgroup`, `table` or `treegrid`:</p>
               <ul>
-                <li>
-                  <a data-cite="wai-aria-1.1#aria-colindex">`aria-colindex`</a>
-                </li>
-                <li>
-                  <a title="aria-rowindex" data-cite="wai-aria-1.1#aria-rowindex">`aria-rowindex`</a>
-                </li>
-                <li>
-                  <a data-cite="wai-aria-1.1#aria-selected">`aria-selected` (state)</a>
-                </li>
-                <li>
-                  <a data-cite="wai-aria-1.1#aria-activedescendant">`aria-activedescendant`</a>
-                </li>
+                <li><a data-cite="wai-aria-1.1#aria-activedescendant">`aria-activedescendant`</a></li>
+                <li><a data-cite="wai-aria-1.1#aria-colindex">`aria-colindex`</a></li>
+                <li><a data-cite="wai-aria-1.2#aria-colindextext">`aria-colindextext`</a></li>
+                <li><a data-cite="wai-aria-1.1#aria-disabled">`aria-disabled`</a></li>
+                <li><a data-cite="wai-aria-1.1#aria-rowindex">`aria-rowindex`</a></li>
+                <li><a data-cite="wai-aria-1.2#aria-rowindextext">`aria-rowindextext`</a></li>
+                <li><a data-cite="wai-aria-1.1#aria-selected">`aria-selected` (state)</a></li>
               </ul>
               <p>Also, if child of `role=treegrid`:</p>
```

Comment:
I think we should change "also" to "Only" here, since these are MUST NOTs for non-treegrids